src/macros.rs

```diff
@@ -12,19 +12,20 @@
 /// The following example shows how to load a properly formatted YAML file to build an instance
 /// of an `App` struct.
 ///
-/// ```ignore
-/// # #[macro_use]
-/// # extern crate clap;
-/// # use clap::App;
+/// ```no_run
+/// # #![cfg(feature = "yaml")]
 /// # fn main() {
+/// # use clap::{App, load_yaml};
 /// let yml = load_yaml!("app.yml");
 /// let app = App::from_yaml(yml);
 ///
 /// // continued logic goes here, such as `app.get_matches()` etc.
 /// # }
+/// # #![cfg(not(feature = "yaml"))]
```

Comment:
This line seems to be involved in breaking CI. You can't have a `#![...]` attribute on a function, it needs to be a `#[...]` attribute instead.